### DISCUSSION TITLE
Embedding Iframe

### DISCUSSION THREAD

**Discussion by i-KishoreVarma** [facebook/docusaurus#7984] [Q&A]
Lets say i have the following structure in /docs/Project/

```
.
├── project.mdx
└── reports
    ├── index.html
    ├── lib
    └── report.html
```

In project.mdx i am using iframe like this

`<iframe src='./reports/report.html' width='100%'></iframe>`

I am getting page not found in iframe.

**Comment by slorber** [comment]
`/docs/` is for Markdown documents, not for standalone html pages.

You could try something like `/static/reports/report.html` and then link to `<iframe src='/reports/report.html' width='100%'></iframe>`
